xarray/core/duck_array_ops.py

```diff
@@ -171,6 +171,79 @@ def _ignore_warnings_if(condition):
         yield
 
 
+def _nansum(value, axis=None, **kwargs):
```

Comment:
Can we give these functions a more explicit names like `_nansum_object`?

Reply:
Done